### ISSUE TITLE
Ablative improvements from different improvements

### ISSUE THREAD

**Issue by ayushtues** [nyrahealth/CrisperWhisper#28]
Hi, thanks for open-sourcing such an amazing model! 

A few questions on the relative improvements due to different improvements suggested. How much WER improvement did you observe due to each of these changes - 

1. Tokenizer changes ( the space fix wrt the Whisper tokeniser ) 
2. Finetuning on highly curated verbatim datasets
3.  Adding the attention loss ( was this only responsible for improving timestamp accuracy, or WER as well? ) 

Also for step 1, did you freeze the base whisper model, and only fine-tuning the output logit prediction layer which predicts the tokens? 

Thanks!
